wifiphisher/data/wifiphiser-os-initial-requests

```diff
@@ -0,0 +1,9 @@
+Android|google.com/generate_204
+Android|connectivitycheck.gstatic.com/generate_204
+Android|connectivitycheck.android.com/generate_204
+Android|gstatic.com/generate_204
+Apple|captive.apple.com/hotspot-detect.html
+Apple|apple.com/library/test/success.html
+Windows|msftconnecttest.com
+Windows|msftncsi.com
+Linux|detectportal.firefox.com
```

Comment:
I believe this is a Firefox URI and not a Linux generic.

Reply:
it would only work if we only compared the first request, removed